lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -358,10 +358,10 @@ class WP_HTML_Tag_Processor {
 	/**
 	 * Which class names to add or remove from a tag.
 	 *
-	 * These are tracked separately from attribute updates because they are
-	 * semantically distinct, whereas this interface exists for the common
-	 * case of adding and removing class names while other attributes are
-	 * generally modified as with DOM `setAttribute` calls.
+	 * These are tracked separately from other lexical updates because
```

Comment:
Does this comment need to change? It seems clearer to me before the change because we're not comparing class name updates to lexical updates, but we're directly contrasting how class names are handled differently than attribute updates. Or in other words, this is an answer to the question, "why aren't class names treated the same way as attributes since they are HTML attributes?"

No strong feelings, but maybe if we bring in "lexical updates" here we might want to rethink what we want to communicate with the paragraph.

Reply:
Yeah, I wasn't 100% sure if this should be changed myself. It's true that while we're renaming to `$lexical_updates` to prepare them to hold non-attribute updates, they don't do that just yet, so we might as well keep the PHPDoc unchanged; and if we end up re-introducing a dedicated `$attribute_updates` field (like in #47068), the unchanged PHPDoc would continue to apply (since we'd once again keep class name updates separate from attribute updates).

So I guess I'll revert 👍

Reply:
08d1380